selenium_tests/lti_emailer/emailer_is_loaded_tests.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium_tests.lti_emailer.page_objects.emailer_mainpage_page_object import EmailerMainPage
+
+
+class EmailerIsLoadedTests(EmailerBaseTestCase):
+
+    def test_tool_is_loaded_on_page(self):
```

Comment:
I was not expecting this to be a separate file, but an additional tests within emailer_permission_tests.py.  Is the intention to add more is_loaded tests within this file?